core/app/[locale]/(default)/account/(tabs)/_components/message-wrapper.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+
+import { Message } from '~/components/ui/message';
+
+import { useAccountStatusContext } from './account-status-provider';
+
+interface Props {
+  message?: string;
+}
+
+export const MessageWrapper = ({ message }: Props) => {
```

Comment:
What do you think about renaming it? Smth like  `Notification`, `AccountNotification` etc...

Reply:
I think `AccountNotification` might be a good option, thanks!